src/npe2/_inspection/_from_npe1.py

```diff
@@ -377,6 +377,9 @@ def napari_get_writer(self, impl: HookImplementation):
         )
         return NotImplemented  # pragma: no cover
 
+    def napari_write_graph(self, impl: HookImplementation):
+        self._parse_writer(impl, "graph")
```

Comment:
Is this function required?

I don't recall `napari_plugin_engine` having a `napari_write_graph` callback.

If it _is_ required, it would be nice to have it tested. Adding the hook to the test npe1 plugin should do the trick. See `tests/npe1-plugin/npe1_module/__init__.py` for where to add a `@napari_hook_implementation` stub.

Reply:
@jwindhager could you take a look at this?
This change came from your PR.

Reply:
Hi @nclack and thanks for pinging me @JoOkuma 

You're right of course, the npe1 obviously doesn't have such a callback and this is therefore dead code. I assume we won't backport support for graph plugins to the npe1? In this case, we can just remove the function.

```suggestion
```

Reply:
Thanks @jwindhager!

I went ahead and accepted the suggestion (hope that was ok @JoOkuma). I'm not aware of any plans to backport.  If I'm wrong though, we can always add this back.